requirements-dev.txt

```diff
@@ -0,0 +1,5 @@
+-r requirements.txt
```

Comment:
No question is stupid. I had to install the sphinx dependencies to be able to build the docs locally. Also I remember that you use nose for running test, no?